data/35e/wizards_of_the_coast/core/universal_rules/ur_templates.lst

```diff
@@ -248,15 +250,16 @@ Assign Subtype ~ Human							RACESUBTYPE:Human
 
 
 ###Block:
-Level Adjustment +1	LEVELADJUSTMENT:1
-Level Adjustment +2	LEVELADJUSTMENT:2
-Level Adjustment +3	LEVELADJUSTMENT:3
-Level Adjustment +4	LEVELADJUSTMENT:4
-Level Adjustment +5	LEVELADJUSTMENT:5
-Level Adjustment +6	LEVELADJUSTMENT:6
-Level Adjustment +7	LEVELADJUSTMENT:7
-Level Adjustment +8	LEVELADJUSTMENT:8
-Level Adjustment +9	LEVELADJUSTMENT:9
+# This block is repeated above ?
```

Comment:
Before we comment out an entire block, perhaps we should dig in a little. This section is for manually adding a Level Adjustment. The above is not. You can delete the repeat hidden above, but I would not delete the general homebrew adjustments here.

Reply:
ok, this bit can be backed out, am surprised that PCGen doesnt flag an error with loading duplicate templates though.